src/main/java/dave/Dave.java

```diff
@@ -0,0 +1,63 @@
+package dave;
+
+import java.io.IOException;
+
+import dave.commands.Command;
+import dave.exceptions.ChatbotException;
+
+public class Dave {
+    /** The output file. */
+    private Storage storage;
+    /** The task list to record tasks. */
+    private TaskList taskList;
+    /** The UI to print feedback to user. */
+    private Ui dave;
```

Comment:
I agree, daveUi would separate it from the the class dave's instance and avoid confusion